src/pages/utilities/UnrecognizedUtilityTabs/LinkFilesTab.tsx

```diff
@@ -151,8 +151,8 @@ function LinkFilesTab() {
     }
   }), [setLinks]);
 
-  const removeLink = useCallback((fileId: number) => setLinks((linkState) => {
-    const itemIndex = linkState.findLastIndex(link => link.FileID === fileId);
+  const removeLink = useCallback((FileID: number) => setLinks((linkState) => {
+    const itemIndex = linkState.reverse().findIndex(link => link.FileID === FileID);
```

Comment:
What was the issue with `findLastIndex`?

Reply:
I think this was pulled in from @revam's stuff, I only changed class names.

Reply:
Never mind, it was from changes @hidden4003 made.

![image](https://github.com/ShokoAnime/Shoko-WebUI/assets/9443295/e26e2ee2-cfd7-4358-ae40-8c3c7dd6840f)

Reply:
I don't remember changing that

Reply:
You probably want to revert this piece so it doesn't pop up in the diff if you didn't change it, as it looks like it tries to revert to some previous version or smth.